src/main/java/org/apache/commons/text/RandomStringGenerator.java

```diff
@@ -183,9 +183,11 @@ public Builder filteredBy(final CharacterPredicate... predicates) {
          * @since 1.2
          */
         public Builder selectFrom(final char... chars) {
-            characterList = new ArrayList<>();
-            for (final char c : chars) {
-                characterList.add(c);
+            if (chars != null) {
+                characterList = new ArrayList<>();
```

Comment:
-1: This PR breaks the documented contract in the Javadoc (the contract is still broken for null). This is unexpected because the consequence of null input should be the same as empty (size 0) input but in fact, is not. 
For example:
```
builder.
  selectFrom('a')
  selectFrom();
```
gives us a `characterList` of `[]` but:
```
builder.
  selectFrom('a')
  selectFrom(null);
```
gives us a `characterList` of `['a']` instead of `[]`.
IOW, null input and empty should be the same. If not, why not? Or am I missing something? Why doesn't `selectFrom()` always reset `characterList`?